### ISSUE TITLE
Project status and relation to ezdof2

### ISSUE THREAD

**Issue by buhtz** [T0ha/ezodf#25]
Hi. Do you know about this?
https://github.com/iwschris/ezodf2

The last commit there was 2013. But the README says that it is derived from "ezdof".
But your last commit is much more fresh. ;)

So can you give some more infos about it? I would assume that your "ezdof" is newer and more stable then "ezdof2"?

**Comment by chfw** [comment]
Yep, I am aware of that. This project newer than ezodf2. However, I started my fork and is about to make a new release(under pyexcel-ezodf) since ezodf 0.3.2. 

Please read this note too: https://github.com/pyexcel/pyexcel-ezodf/issues/2

**Comment by buhtz** [comment]
Ah complex relations. It would be nice if this could be described in details in all three (or more) related projects.

Do I understand you right?
`pyexcel-ezodf` is currently the newsest, freshes and most active (supported) project?
`ezodf2` is absolute outdated (last commit 2013)?
And `ezodf` is active, newer then `ezodf2` but out-dated in relation to `pyexcel-ezodf`?

**Comment by chfw** [comment]
I have mentioned recent move in https://github.com/pyexcel/pyexcel-ezodf#maintenance.  And I am trying to refresh the library in pypi using package name pyexcel-ezodf but it won't have any impact on programming level. Still you do 'import ezodf'.

**Comment by buhtz** [comment]
I think I plan to `import ezodf`.

Does T0ha told he just make "holiday" and will come back? From my point of view it would have been nice if he would let you join his project/repository and you work with that. I would appreciate if you booth would re-join your repositories.

**Comment by chfw** [comment]
I agree that it will be nice I would join. However, I got to roll out the fix for https://github.com/alphagov/notifications-admin/pull/1426.  I don't think I would wait for [another year](https://github.com/T0ha/ezodf/pull/18). However, I could always upstream my changes in pyexcel-ezodf as long as someone click accept the PR.

**Comment by buhtz** [comment]
I see your point. What I meant was that t0ha should give you admin rights to his project. Than you can merge the PR yourself. ;) I think this is no big deal.
